core/base/configmaps.yaml

```diff
@@ -33,3 +33,10 @@ kind: ConfigMap
 apiVersion: v1
 metadata:
   name: qeb-hwt-workflow
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: pulp-python-index
```

Comment:
noticed that we have named everything with prefix `pulp-pypi` , maybe it would be good to have consistency
and name the configmap and secret as  `pulp-pypi`.

Reply:
Good point, Adjusted. 👍🏻